contracts/managers/allocators/BinaryAllocator.sol

```diff
@@ -497,4 +507,21 @@ contract BinaryAllocator is
             )
```

Comment:
What do you think about the `calculateCollateralIDHashFromSet` function being public?